### PR TITLE
Pods 7694

### PR DESCRIPTION
**Pull request by jamdade-sharad**

### CODE REVIEW

File: app/controllers/common/MembersSummaryController.scala [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+          key = memberSummary.name,
+          firstValue = memberSummary.nINumber,
+          secondValue = currencyFormatter.format(memberSummary.PaymentValue),
+          actions = Some(Actions(
+            items = Seq(
+              ActionItem(
+                content = Text(Message("site.view")),
+                href = eventType match {
+                  case Event22 => controllers.event22.routes.Event22CheckYourAnswersController.onPageLoad(index).url
+                  case Event23 => controllers.event23.routes.Event23CheckYourAnswersController.onPageLoad(index).url
```
Comment (grantrobson) [inline app/controllers/common/MembersSummaryController.scala:85]: You need a case _ => here

File: app/controllers/event1/UnauthPaymentSummaryController.scala [hunk 2]
```diff
@@ -39,24 +40,42 @@ import views.html.event1.UnauthPaymentSummaryView
...
+    Ok(view(form, waypoints, mappedMemberOrEmployer, sumValue(request.userAnswers)))
+  }
 
-      val value = ValueViewModel(HtmlFormat.escape(memberOrEmployerSummary.unauthorisedPaymentValue.toString()).toString)
+  private def sumValue(userAnswers: UserAnswers) =
+    currencyFormatter.format(userAnswers.sumAll(MembersOrEmployersPage(EventType.Event1), MembersOrEmployersSummary.readsMemberOrEmployerValue))
+
+  def onSubmit(waypoints: Waypoints): Action[AnyContent] = (identify andThen getData(EventType.Event1) andThen requireData) {
+    implicit request =>
+      val mappedMemberOrEmployer = getMappedMemberOrEmployer(request.userAnswers)
```
Comment (grantrobson) [inline app/controllers/event1/UnauthPaymentSummaryController.scala:64]: Move this down into formWithErrors big or change to a def/ lazy val

File: app/controllers/common/MembersSummaryController.scala (hunk 1 above)
Comment (jamdade-sharad) [reply app/controllers/common/MembersSummaryController.scala:85]: default case added

File: app/controllers/event1/UnauthPaymentSummaryController.scala (hunk 2 above)
Comment (jamdade-sharad) [reply app/controllers/event1/UnauthPaymentSummaryController.scala]: moved under formWithErrors

**Review by grantrobson — APPROVED** [review]

**Pull request merged**